activiti-api/activiti-task-model-api-impl/src/main/java/org/activiti/runtime/conf/TaskPayloadMixIn.java

```diff
@@ -0,0 +1,16 @@
+package org.activiti.runtime.conf;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+
+import static com.fasterxml.jackson.annotation.JsonTypeInfo.As.PROPERTY;
+import static com.fasterxml.jackson.annotation.JsonTypeInfo.Id.NAME;
+
+@JsonTypeInfo(
+        use = NAME,
+        include = PROPERTY,
+        property = "payloadType")
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class TaskPayloadMixIn {
```

Comment:
This should probably be merged with `ProcessPayloadMixIn` and moved to common.

Reply:
@erdemedeiros look comment above